src/folsom_ets.erl

```diff
@@ -397,6 +397,13 @@ notify(Name, Value, histogram, false) ->
     add_handler(histogram, Name),
     folsom_metrics_histogram:update(Name, Value),
     ok;
+notify(Name, Value, {histogram, SampleType}, true) ->
```

Comment:
Is SampleType supposed to be unused in this case?  If so, you may want to add a leader `_` to the variable name.